README.md

```diff
@@ -12,34 +19,38 @@ If you are want to create a search UI for your website then checkout our [React
 * [License](#license)
 
 ## Install
+
+### NPM/Yarn
+
+```
+npm install --save @sajari/sdk-js
 ```
-npm install --save sajari
+
+### Browser
+
+```html
+<script src="https://unpkg.com/@sajari/sdk-js@1.0.0/dist.iife/main.js"></script>
 ```
 
 ## Getting Started
 
-A quick search example using the `website` search pipeline.
+Here's a quick example that searches for a webpage.
 
 ```javascript
-import { Client, Tracking } from "sajari";
+import { Client, BaseSession, TextSession, TrackingClick } from "@sajari/sdk-js";
 
-const client = new Client("<project>", "<collection>");
+const client = new Client("sajariptyltd", "sajari-com").pipeline("website");
+const session = new TextSession("q", new BaseSession(TrackingClick, "url", {}));
```

Comment:
Can this be easier to setup by default?  It seems annoying to have to put this in everytime.  At least explain what it all means...